package.json

```
@@ -8,7 +8,9 @@
     "start": "next start",
     "lint": "next lint",
     "test": "jest",
-    "test:watch": "jest --watch"
+    "test:watch": "jest --watch",
+    "prepare": "husky",
+    "prettier:check:ci": "npx prettier . --check"
```

```suggestion
    "prettier:check:ci": "npx --no prettier . --check"
```
This will prevent npx from trying to do anything updating.